build.gradle

```
@@ -219,10 +219,13 @@ runClient {
 
 def getVersionMetadata() {
 	def build_id = System.getenv("GITHUB_RUN_NUMBER")
+	def workflow_id = System.getenv("GITHUB_WORKFLOW")
 
 	// CI builds only
-	if (build_id != null) {
-		return "build.${build_id}"
+	if (build_id != null && workflow_id != "build-release") {
+		return "+build.${build_id}"
+	} else if (workflow_id == "build-release") {
+		return ""
 	}
```

Logic simplification
```suggestion
	if (workflow_id == "build-release") {
		return ""
	} else if (build_id != null) {
		return "+build.${build_id}"
	}
```